apps/application-system/api/infra/application-system-api.ts

```diff
@@ -121,6 +121,16 @@ export const serviceSetup = (services: {
       PARTY_LETTER_REGISTRY_API_BASE_PATH: ref(
         (h) => `http://${h.svc(services.servicesPartyLetterRegistryApi)}`,
       ),
+      DRIVING_LICENSE_PATH_REPLACEMENT_FROM: {
+        prod: '/RafraentOkuskirteini-v1/api/Okuskirteini/',
+        staging: '',
```

Comment:
Shouldn't these be also for all envs?

Reply:
well, there's no staging environment for this api, and prod is the only place we need to do this horrible hack ..

Reply:
perhaps, i should have just changed this to use the `isRunningOnEnvironment` helper.. and not bother with any env vars.. this should also be a short-term change, apparently

Reply:
No, please do not use that helper.